rosa_hcp/rosa-hcp-quickstart-guide.adoc

```diff
@@ -37,14 +38,40 @@ The Terraform instructions are for testing and demonstration purposes. Your own
 ====
 
 [discrete]
-include::modules/rosa-hcp-vpc-terraform.adoc[leveloffset=1]
+include::modules/rosa-hcp-create-network.adoc[leveloffset=+3]
 
 [role="_additional-resources"]
-[id="additional-resources_rosa-hcp-quickstart-vpc-terraform"]
+[id="additional-resources_rosa-hcp-create-network-quickstart"]
+.Additional resources
+
+* See the link:https://aws.amazon.com/cloudformation/[AWS CloudFormation] for more information about structuring CloudFormation files to create VPCs.
```

Comment:
Is it worth linking directly to https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/quickref-ec2-vpc.html ?